app/jobs/story_update_job.rb

```diff
@@ -20,7 +20,7 @@ def receive_story_update(data)
     load_resources(data)
     episode ? update_episode : create_episode
```

Comment:
Sounds good.  I guess i just want this method to silently do nothing if (1) the story has no series, or (2) the story-series isn't in feeder.  Which feel like non-errors.  And currently it'll throw in either case, which will probably reach newrelic.